app/forms/idv/ipp/address_form.rb

```diff
@@ -0,0 +1,38 @@
+module Idv
```

Comment:
I find it kind of confusing how we have a mix of `/idv/ipp/...` and `/idv/in_person/...` (this one mainly for views) directory structures. Thoughts on consolidating to `/idv/ipp/...` in a future PR?

Reply:
Good catch! I'm a big fan of clearer names and fewer acronyms so if we were going to do a mass rename, i'd lean on `InPerson` rather than `Ipp` but I don't feel that strongly

Reply:
+1 `InPerson`

Reply:
`in_person` / `InPerson` it is! I'll create a ticket to do this rename